tests/lua/lua-transform-01/transform.lua

```diff
@@ -0,0 +1,10 @@
+-- Arguments supported
+local bytes_key = "bytes"
+local offset_key = "offset"
```

Comment:
Do these have meaning in this script?

Reply:
@jlucovsky ^

Reply:
No ... I'm preparing a branch with updates and will address this.